app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb

```diff
@@ -0,0 +1,73 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach religious education</h2>
+    <div class="inset">
+      <p>Through teaching religious education, you’ll help pupils gain a deeper awareness of the world around them by promoting empathy and understanding. By investigating religious and non-religious views, you’ll give pupils a chance to reflect on their own beliefs and articulate their principles.</p>
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching religious education opens the door for your pupils to pursue exciting careers. From humanitarian work and charity work, to the civil service and journalism.</p>
+      <p>Your pupils will learn more from you than you might think. As their religious education teacher, you'll guide your pupils through the skills they need to show their understanding of the rich diversity of beliefs and to engage with big ideas with respect. You will teach them valuable life skills such as empathy, critical thinking and deep reflection.</p>
```

Comment:
`to engage **with** big ideas **with** respect` is slightly repetitive so slightly harder to read - is it worth tweaking the wording slightly?